drs_filer/api/20200622.383121d.data_repository_service.swagger.yaml

```diff
@@ -22,6 +22,37 @@ produces:
 #  - {}
 #  - authToken: []
 paths:
+  '/objects':
+    get:
+      summary: List objects.
+      description: Retrieve a list of DRS objects.
+      operationId: GetAllObjects
+      responses:
+        '200':
+          description: A list of DRS objects.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/DrsObject'
+        '400':
+          description: The request is malformed.
+          schema:
+            $ref: '#/definitions/Error'
```

Comment:
I recently realized that it is actually impossible to send a `BadRequest` to an endpoint, when all you do is specify a method (`GET`) and an endpoint (`/objects`) and adding additional parameters is not explicitly forbidden (which it is not):

If the endpoint is wrong or the method is not defined for the endpoint, there will anyway be a different error message (but not raised by your controller, which is never called). So in my opinion (and I have seen this elsewhere), we can remove the `400` here.

It's a different story if the endpoint/operation has params, like path or query params. These could be used wrongly, in which case a `400` is warranted.